cmd/autoconfigbrancher/helper/helper.go

```diff
@@ -0,0 +1,24 @@
+package helper
+
+import (
+	"os/exec"
+)
+
+func Call(cmd string, args ...string) ([]byte, []byte, error) {
+	stdout := &OutputWriter{}
+	stderr := &OutputWriter{}
+	c := exec.Command(cmd, args...)
+	c.Stdout = stdout
+	c.Stderr = stderr
+	e := c.Run()
+	return stdout.output, stderr.output, e
```

Comment:
I think you can just use `os.Cmd.Output()` or `CombinedOutput()`

Reply:
Done.